.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -1,15 +1,16 @@
-name: Task/Feature request
-description: Filing a Task/Feature request
-title: "[Task]: "
-labels: ["type:feature", "needs:info", "repository:addons-server", "repository:addons-frontend"]
+name: Feature request
+description: Filing a Feature request
+title: "[Feature]: "
```

Comment:
For most of our tickets `Task` describes more accurately than `Feature` the type of work being done... Could we keep it ? And maybe just say `item 1` instead of `task 1` below ?